crates/bevy_core_pipeline/src/core_3d/mod.rs

```diff
@@ -307,7 +307,7 @@ pub fn prepare_core_3d_depth_textures(
                     dimension: TextureDimension::D2,
                     // PERF: vulkan docs recommend using 24 bit depth for better performance
                     format: TextureFormat::Depth32Float,
-                    usage,
+                    usage: camera_3d.depth_texture_usages.into(),
```

Comment:
I think this PR was before the prepass was merged, it should probably be combined with the usage above to keep compatibility with the prepass because right now I expect it to break

Reply:
fixed